sim/scenarios/blackhole/blackhole-error-model.cc

```diff
@@ -0,0 +1,27 @@
+#include "blackhole-error-model.h"
+ 
+NS_OBJECT_ENSURE_REGISTERED(BlackholeErrorModel);
+ 
+TypeId BlackholeErrorModel::GetTypeId(void) {
+  static TypeId tid = TypeId("BlackholeErrorModel")
+    .SetParent<ErrorModel>()
+    .AddConstructor<BlackholeErrorModel>()
+    ;
+  return tid;
+}
+ 
+BlackholeErrorModel::BlackholeErrorModel() : enabled_(true) { }
+ 
+bool BlackholeErrorModel::DoCorrupt(Ptr<Packet> p) {
```

Comment:
does this cause the packet to get dropped or corrupted?

Reply:
It just gets dropped. You could corrupt the packet by modifying the raw packet contained in `p`.